src/app/active-mode/Views/PursuitsView.tsx

```diff
@@ -84,14 +84,13 @@ function PursuitsView({ store, trackedTriumphs, defs, profileResponse }: Props)
     >
       <div className={styles.activePursuits}>
         {pursuits.sort(sortPursuits).map((item) => (
-          <Pursuit item={item} key={item.index} defs={defs!} hideDescription={true} />
+          <Pursuit item={item} key={item.index} hideDescription={true} />
```

Comment:
The only bit that gives me a bit of a tinge is if you're always using a component somewhere that already has defs available, it'd be nice to just pass them through.

Reply:
this describes a lot of places unfortunately, which is why there's sooooo many exclamation points here. if this change isn't a performance problem, afterward i would love to standardize where the "if defs" wall exists near the top of the whole tree, and have an assumptive non-null selector for almost everywhere

Reply:
this is an unrelated thought i guess, just to clarify